Tests/TestsCommon/CommonTestUtils.swift

```diff
@@ -0,0 +1,172 @@
+import Foundation
```

Comment:
Everything here used to be in `TestUtils.swift`. All that has changed is marking things open and public as needed so they can be accessed from the other test targets.